pillar/fluentd/server_operations_qa.sls

```diff
@@ -95,10 +89,11 @@ fluentd:
                 directive_arg: '**'
                 attrs:
                   - '@id': es_outbound
-                  - '@type': elasticsearch_dynamic
+                  - '@type': elasticsearch
```

Comment:
Does the newer elasticsearch builtin support the dynamic functionality now? We had used that plugin to allow for programmatically generating the logstash_prefix.

Reply:
Reverted, but looks like we should be maybe handling things on the ES side instead based on their [docs](https://github.com/uken/fluent-plugin-elasticsearch#dynamic-configuration)